From b18c6c48e4dabaf8bf9b178b3deca631fc79c96e Mon Sep 17 00:00:00 2001
From: Petr Vorel <pvorel@suse.cz>
Date: Tue, 11 Jun 2019 21:51:58 +0200
Subject: [PATCH] traceroute6: fix build with NLS

commit 418d8be ("build-sys: fix build with NLS") fixed problems for
default packages using libintl. Fix also traceroute6 (build it must be
enabled with -DBUILD_TRACEROUTE6=true)

Signed-off-by: Petr Vorel <pvorel@suse.cz>
Upstream status: 81bfb31a714641b70c2a7c7d9ce12cf08f98a372
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 2b545b6..85db8a6 100644
--- a/meson.build
+++ b/meson.build
@@ -266,7 +266,7 @@ endif

 if build_traceroute6 == true
	executable('traceroute6', ['traceroute6.c', git_version_h],
-		dependencies : [cap_dep, idn_dep],
+		dependencies : [cap_dep, intl_dep, idn_dep],
		link_with : [libcommon],
		install: true)
	meson.add_install_script('build-aux/setcap-setuid.sh',
--
2.21.0
